### DISCUSSION TITLE
Seeing calls to __data.js after upgrade from next.431 to next.456

### DISCUSSION THREAD

**Discussion by j4w8n** [sveltejs/kit#6457] [General]
Not complaining. Just curious.
I searched the Discussions and open/closed Pulls for `__data.js` and couldn't find anything.

I'm in troubleshooting mode for some other things, so I've been logging `event.url.pathname` in `handle()`.
`<path>/__data.js` seems to show up whenever I use `invalidate()`; so I assume this is helping to update page data?

**Comment by kristjanmar** [comment]
Yes I've just realised this, too. It seems like __data.json has been replaced with __data.js.

I have Cloudflare on my site and they cache .js files given that they are usually static. Now my page props that are supposed to be dynamic are being cached on the edge.

Edit: Looks like this was introduced in version 447 with #6318.

**Comment by kristjanmar** [comment]
I created an issue for this since I think this can cause problems for anyone using Cloudflare #6458

**Reply by laxadev** [reply]
@kristjanmar I'm facing the same problems. On the first load, I can only see old content, after refreshing the site the new content shows up too.
Do you have a workaround for this?
I have tried to set Page rules on CF end but it doesn't seem to be helping.
https://domain.com/*__data.js* with Browser Cache TTL: 2 minutes
Also, I tried Cache Level Bypass mode but had no luck.

**Reply by kristjanmar** [reply]
I ended up rolling back to version 446 for now, the last version before it was changed to __data.js.

What about if you try setting the page rule as Cache Level: Bypass?

`*domain.com/*__data.js --> Cache Level: Bypass`

I believe that should work to at least make Cloudflare ignore the file. If you really need the browser cache to be set then you should be able to do that in SvelteKit by setting `Cache-Control: public, max-age: 120`

I've been using hooks.js to set cache control headers but there may be a way to do it in the load function now.
